src/OpenTracing.Contrib.Grpc/Extensions.cs

```diff
@@ -35,5 +36,27 @@ public static string ToReadableString(this Metadata metadata)
 
             return string.Join(";", metadata.Select(e => $"{e.Key} = {e.Value}"));
         }
+
+        public static string ToReadableString(this CallOptions options)
+        {
+            // Should this be converted to milis?
+            var deadline = options.Deadline.HasValue ? options.Deadline.Value.ToUniversalTime().ToString(CultureInfo.InvariantCulture) : "Infinite";
```

Comment:
I made it `InvariantCulture` to avoid differences between systems.

Reply:
Good catch, thanks!